### ISSUE TITLE
Ilastik in the Cloud

### ISSUE THREAD

**Issue by lmiroslaw** [ilastik/ilastik#1719]
This is a small project that shows how to integrate Ilastik with Azure Batch. The idea is that user can just submit the computing job to Azure and download the results when the calculation is done. Azure Batch spins up a cluster, runs the ilastik in a scallable manner and shuts down the cluster, when the job is complete.

[github](https://github.com/lmiroslaw/azure-batch-ilastik)

**Comment by k-dominik** [comment]
Hey @lmiroslaw,

your project looks awesome! Thanks a lot for figuring out the azure cloud deployment with ilastik! I haven't tried your scripts yet, but looking forward to that.

**Comment by lmiroslaw** [comment]
That would be great if you could test it and share the feedback with us!

**Comment by wolny** [comment]
Hi @lmiroslaw, great job getting ilastik to work with Azure Batch. I've tested your scripts with my free azure subscription and it all works perfectly fine.
You've picked up a really nice sample dataset (Drosophila stack) for the demo purposes. In order show a more realistic scenario of how one can use ilastik to process large bio-image stacks in fully parallel manner on azure batch, I've build a simple app based on your idea of Drosophila nuclei segmentation. Please see my PR https://github.com/lmiroslaw/azure-batch-ilastik/pull/2 for more details.
*tl;dr* instead of processing the whole Drosophila stack in a single task (like in your example) I've used ilastik's `cutout_subregion` option in order to process a given timepoint 3D volume in a separate task.
The task's command simply becomes:
```
./run_ilastik.sh --headless --project=pixelClassification.ilp --export_source="Simple Segmentation" --output_format="multipage tiff" --output_filename_format="{nickname}_{1}_seg.tiff" --export_dtype="uint16" --cutout_subregion="[({t}, 0, 0, 0, 0), ({t+1}, 300, 275, 50, 2)]" drosophila_00-49.h5
```
The app spins up one task per each time point (50 in total) and then downloads the results.

Please let me know if you have any questions regarding my PR or otherwise consider merging it into your master branch and updating the README, so that we could spread the word among our python-loving ilastik community :grin: Cheers!
